com.yacreader.YACReader.yml

```diff
@@ -0,0 +1,123 @@
+app-id: com.yacreader.YACReader
+runtime: org.kde.Platform
+runtime-version: "5.15-22.08"
+sdk: org.kde.Sdk
+command: YACReader
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=network
+  - --device=dri
+  - --filesystem=home
+cleanup:
+  - /bin/gn
+  - /include
+  - /lib/cmake
+  - /lib/pkgconfig
+  - /lib/systemd
+  - /share/doc
+  - /share/man
+  - "*.a"
+  - "*.la"
+modules:
+  - name: yacreader
+    # https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=yacreader
+    buildsystem: qmake
+    config-opts:
+      - CONFIG+=unarr pdfium server_bundled
+    install-rule: null
+    make-install-args:
+      - sub-YACReader-install_subtargets
+      - sub-YACReaderLibrary-install_subtargets
+    post-install:
+      - mv "$FLATPAK_DEST"/share/applications/{YACReader,com.yacreader.YACReader}.desktop
+      - mv "$FLATPAK_DEST"/share/applications/{YACReaderLibrary,com.yacreader.YACReader.YACReaderLibrary}.desktop
+      - mv "$FLATPAK_DEST"/share/icons/hicolor/scalable/apps/{YACReader,com.yacreader.YACReader}.svg
+      - mv "$FLATPAK_DEST"/share/icons/hicolor/scalable/apps/{YACReaderLibrary,com.yacreader.YACReader.YACReaderLibrary}.svg
+      - sed -i -e 's/^Categories=\(.*\)$/Categories=Office;\1/' -e 's/^Icon=.*$/Icon=com.yacreader.YACReader/' "$FLATPAK_DEST"/share/applications/com.yacreader.YACReader.desktop
+      - sed -i -e 's/^Categories=\(.*\)$/Categories=Office;\1/' -e 's/^Icon=.*$/Icon=com.yacreader.YACReader.YACReaderLibrary/' "$FLATPAK_DEST"/share/applications/com.yacreader.YACReader.YACReaderLibrary.desktop
```

Comment:
instead of `sed` please use `destkop-file-edit`

Also with the rename icon suggested above, it's taken care of witht the main .desktop file.

Reply:
Done - rename for YACReader, desktop-file-edit for YACReaderLibrary.